src/NzbDrone.Core/Notifications/NotificationService.cs

```diff
@@ -9,21 +9,32 @@
 using NzbDrone.Core.Qualities;
 using NzbDrone.Core.ThingiProvider;
 using NzbDrone.Core.Tv;
+using NzbDrone.Core.HealthCheck;
+using NzbDrone.Core.HealthCheck.Checks;
+using NzbDrone.Core.Update;
 
 namespace NzbDrone.Core.Notifications
 {
     public class NotificationService
         : IHandle<EpisodeGrabbedEvent>,
           IHandle<EpisodeDownloadedEvent>,
-          IHandle<SeriesRenamedEvent>
+          IHandle<SeriesRenamedEvent>,
+          IHandleAsync<HealthCheckCompleteEvent>
     {
         private readonly INotificationFactory _notificationFactory;
         private readonly Logger _logger;
+        private readonly IHealthCheckService _healthCheckService;
+        private readonly ICheckUpdateService _checkUpdateService;
 
-        public NotificationService(INotificationFactory notificationFactory, Logger logger)
+        private UpdatePackage _lastUpdate;
```

Comment:
Thats kinda how couch and sickbeard worked, so I decided to go with that behavior. It was either an oversight, or designed so you got notifications again if you missed them the first time.

What would be better?